app/ancillaryFactory/src/components/NoAncillaryMessage.js

```diff
@@ -0,0 +1,15 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, Translation } from '@kiwicom/mobile-shared';
+
+const NoAncillary = () => (
+  <View>
+    <Text>
+      <Translation passThrough="No ancillary view found for the provided parameters." />
```

Comment:
Shouldn't this also be translated?

Reply:
I think no, I added this component as a helper for developers if they mistype ancillary service name.